package.json

```diff
@@ -101,26 +101,30 @@
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "test": "npm run compile && node ./out/test/runTest.js"
+        "test": "npm run compile && node ./out/test/runTest.js",
+        "ui-test": "npm run compile && extest setup-and-run out/ui-test/allTestsSuite.js -c 1.40.0"
     },
     "devDependencies": {
-        "typescript": "^2.6.1",
-        "@types/vscode": "1.19.0",
-        "vscode-test": "^1.2.3",
-        "tslint": "^5.8.0",
+        "@types/glob": "^7.1.1",
         "@types/node": "^7.0.43",
+        "@types/chai": "^4.2.0",
         "@types/mocha": "^5.2.5",
-        "mocha": "^5.2.0",
-        "mocha-jenkins-reporter": "^0.4.1",
+        "@types/vscode": "1.19.0",
         "@types/request-promise-native": "^1.0.15",
+        "chai": "^4.2.0",
+        "mocha": "^6.1.4",
+        "mocha-jenkins-reporter": "^0.4.1",
         "glob": "^7.1.6",
-        "@types/glob": "^7.1.1"
+        "tslint": "^5.17.0",
+        "typescript": "^3.5.2",
+        "vscode-test": "^1.2.3",
+        "vscode-extension-tester": "^2.2.0"
     },
     "dependencies": {
         "@types/request": "^2.47.0",
         "@types/yauzl": "^2.9.0",
         "request": "^2.47.0",
-        "request-promise-native": "^1.0.7",
-        "yauzl": "^2.10.0"
+        "yauzl": "^2.10.0",
```

Comment:
why changing the order?
Seems that the previous alphabetic order was better.

Reply:
this was not really intentional. Fixing.